apps/api/src/nodes/3d/types.ts

```diff
@@ -0,0 +1,18 @@
+/** GeoTIFF metadata structure for JSON output */
+export interface GeoTiffMetadata {
```

Comment:
I think we should move this in the GeoTiffMetadataReader node and keep it private.